benchmarks/synchrobench/src/main/java/com/yahoo/oak/synchrobench/contention/abstractions/BenchOp.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak.synchrobench.contention.abstractions;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * The operations that 'CompositionalMap' supports
+ */
+public enum BenchOp {
+    PUT,
+    PUT_IF_ABSENT,
+    PUT_IF_ABSENT_COMPUTE_IF_PRESENT,
+    REMOVE,
+    GET,
+    ASCEND,
```

Comment:
Mention ASCEND and DESCEND is about scan

Reply:
:+1: